dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM php:8.1-apache-bullseye
+
+RUN docker-php-ext-install mysqli sockets pdo pdo_mysql
+
+COPY ./html /var/www/html/
+
+COPY ./apache/000-default.conf /etc/apache2/sites-available/
```

Comment:
Can you explain, why you're using `sites-available/` and not `sites-enabled/ `? In my undestanding, sites added to `sites-available/` have to manually be enabled using `a2ensite` to work.

Reply:
Aaaaaaaaaaaaaaaaaaaah, there is already a symlink to sites-enabled/ by default.

Nice.